client/extensions/woocommerce/state/sites/settings/email/test/helpers.js

```diff
@@ -0,0 +1,72 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { checkEmails } from 'woocommerce/app/settings/email/email-settings/components/helpers';
+
+describe( 'helpers', () => {
+	describe( '#checkEmails', () => {
+		test( 'should return error false on empty string.', () => {
+			expect( checkEmails( '' ) ).to.eql( { error: false } );
+		} );
+		test( 'should return error false on valid email.', () => {
+			expect( checkEmails( 'a@a.com' ) ).to.eql( { error: false } );
+		} );
+		test( 'should return error false on valid comma separated emails.', () => {
+			expect( checkEmails( 'a@a.com, b@b.com' ) ).to.eql( { error: false } );
+		} );
+		test( 'should return error false on valid email with trailing comma.', () => {
+			expect( checkEmails( 'a@a.com,' ) ).to.eql( { error: false } );
+		} );
+		test( 'should return error comma separations error on missing commas', () => {
+			expect( checkEmails( 'a@a.com b@b.com' ) ).to.eql( {
+				error: true,
+				messages: [
+					{
+						error: true,
+						msg: 'a@a.com b@b.com need to be comma separated.',
+					},
+				],
+			} );
+		} );
+		test( 'should return error on invalid email address', () => {
```

Comment:
Isn't this test the same as the one above?

Reply:
Corrected.